### PR TITLE
tests: test_class_image

### PR DESCRIPTION
**Pull request by mukultaneja**
This PR adds following test cases into test_class_image,
1. testSetImageImport
2. testLastImportLayer
3. testGetLastImportLayer

Resolves/Fixes #562

Signed-off-by: mukultaneja <mukultaneja91@gmail.com>

### CODE REVIEW

File: tests/test_class_image.py [hunk 1]
```diff
@@ -49,8 +51,7 @@ def testGetLayerDiffIds(self):
 
     def testGetLayerObject(self):
         self.image2.load_image()
-        self.assertEqual(
-             self.image2.get_layer_object('123abc'), self.image2.layers[0])
+        self.assertEqual(self.image2.get_layer_object('123abc'), self.image2.layers[0])
```
Comment (rnjudge) [inline tests/test_class_image.py]: This line is longer than 79 characters when it's all on one line. It was broken up before to adhere to PEP8 style standards. Probably should revert this change :)

File: tests/test_class_image.py [hunk 2]
```diff
@@ -18,10 +18,12 @@ def setUp(self):
         '''Test a generic Image class'''
         self.image1 = Image('1234abcd')
         self.image2 = TestImage('5678efgh')
+        self.image3 = TestImage('5679efghi')
```
Comment (rnjudge) [inline tests/test_class_image.py]: Could you change the input sha for test `image3` to something that is 8 characters and slightly more distinguishable from `image2`? Maybe `wxyz0987` or something?

Comment (rnjudge) [reply tests/test_class_image.py]: Or maybe there is a reason you chose to do it this way which I am overlooking :)

**Comment by rnjudge** [comment]
@mukultaneja There are a few changes I requested and a questions I had but overall, your work on this PR will bring Tern's coverage percentage for `classes/image.py` to 99% up from 83%. Nice work!

**Comment by mukultaneja** [comment]
@rnjudge thanks for reviewing the PR. I have updated the code to incorporate all the changes. Please review it again!

**Comment by rnjudge** [comment]
Looks good to me. @nishakm want to take a look to make sure I'm not missing anything?

**Comment by nishakm** [comment]
@mukultaneja Could you squash your two commits into one?

**Comment by mukultaneja** [comment]
@nishakm, Done!

**Comment by nishakm** [comment]
@mukultaneja One more thing: can you remove the Resolves/Fixes from your commit message as there is no issue associated with this PR? Once that is done, I will merge this. Thanks for contributing!

**Comment by rnjudge** [comment]
> @mukultaneja One more thing: can you remove the Resolves/Fixes from your commit message as there is no issue associated with this PR? Once that is done, I will merge this. Thanks for contributing!

@nishakm There is an issue open to increase test coverage: https://github.com/vmware/tern/issues/539

@mukultaneja However, since this PR does not fix the entirety of the issue, please use `Works towards #539` in your commit message.

**Comment by mukultaneja** [comment]
@nishak, @rnjudge I have updated the commit message. Please review it!

**Comment by rnjudge** [comment]
@nishakm ok to merge?

**Comment by nishakm** [comment]
@rnjudge Works for me. Merging.
